### DISCUSSION TITLE
change license

### DISCUSSION THREAD

**Discussion by MrRawes** [MetaMask/eth-phishing-detect#11727] [Ideas]
please change the license from "DON'T BE A DICK PUBLIC LICENSE" to something else, my recomendation is [Apache License 2.0](https://spdx.org/licenses/Apache-2.0.html)

**Comment by trn1ty** [comment]
I've thought about this. I don't think it would be feasible to try to get the OK from so many contributors though.

**Comment by BananaFoDavid** [comment]
Maybe "Pinkeye Promise Not To Act Like A Prick In Public"🤟🤘
